akka-http/src/main/scala/akka/http/scaladsl/unmarshalling/GenericUnmarshallers.scala

```diff
@@ -37,28 +37,47 @@ sealed trait LowerPriorityGenericUnmarshallers {
    * Attempt unmarshalling the entity as as `R` first (yielding `R`),
    * and if it fails attempt unmarshalling as `L` (yielding `Left`).
    *
+   * The either unmarshaller only works with strict entities, so make sure to wrap routes that want to use it with
+   * `toStrictEntity`.
```

Comment:
Maybe explicitly point out that it will fail with an exception if you miss out on this.

Reply:
Done.